airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -2187,19 +2188,17 @@ def _build_concurrent_cursor(
             stream_name=model.name or "", namespace=None
         )
 
-        if model.incremental_sync and stream_slicer:
-            # FIXME there is a discrepancy where this logic is applied on the create_*_cursor methods for
```

Comment:
This comment is doodoo but I think I got confused by [this line](https://github.com/airbytehq/airbyte-python-cdk/blob/cb193aba94f01e16e926da635ca3cd6c1380d6a9/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py#L2223) which I fixed [here](https://github.com/airbytehq/airbyte-python-cdk/pull/690/files#diff-1d9bc4ca384e5c00867da05e9db9d919aba908c59bb73aab83883b9ed20def05R2222)